### PR TITLE
Announce impending removal of v1alpha1 API

### PR DESCRIPTION
**Pull request by johngmyers**
The Kubernetes deprecation policy allows removal of alpha APIs with 0 releases and 0 months. Since we have two pending release branches, we might as well give people some warning.

### CODE REVIEW

File: docs/releases/1.16-NOTES.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ the notes prior to the release).
 
 * No required actions yet known.
 
+# Deprecations
+
+* The `kops/v1alpha1` API is deprecated and will be removed in kops 1.18.
```
Comment (zetaab) [inline docs/releases/1.16-NOTES.md]: we need explanation also what this means in case of user. For normal user this does not tell much

Comment (johngmyers) [reply docs/releases/1.16-NOTES.md]: Are there any user considerations? The only thing I can think of is they wouldn't be able to use any v1alpha1 specs with `kops relpace -f`

**Comment by justinsb** [comment]
Thanks @johngmyers - this will be great, despite my protestations ;-)

/approve
/lgtm

**Pull request merged**